x/ugov/msg.go

```diff
@@ -23,3 +25,8 @@ func (msg *MsgGovUpdateMinGasPrice) ValidateBasic() error {
 func (msg *MsgGovUpdateMinGasPrice) GetSigners() []sdk.AccAddress {
 	return checkers.Signers(msg.Authority)
 }
+
+// GetSignBytes implements Stringer method
+func (msg *MsgGovUpdateMinGasPrice) String() string {
+	return fmt.Sprintf("<authority: %s, min_gas_price: %s>", msg.Authority, msg.MinGasPrice.String())
```

Comment:
Is the `<` `>` necessary? I notice before it was only appearing in the dec

Reply:
No, it's just a convention for composed objects.

Reply:
In the future we will have better, automated generator, which doesn't require yaml etc...